messaging/src/main/java/org/axonframework/eventhandling/processors/streaming/pooled/SplitTask.java

```diff
@@ -120,14 +120,19 @@ private boolean splitAndRelease(Segment segmentToSplit) {
                 unitOfWorkFactory
                         .create()
                         .executeWithResult(context -> {
-                            TrackingToken tokenToSplit = tokenStore.fetchToken(name, segmentToSplit.getSegmentId());
+                            TrackingToken tokenToSplit = joinAndUnwrap(
+                                    tokenStore.fetchToken(name, segmentToSplit.getSegmentId(), null)
```

Comment:
```suggestion
                                    tokenStore.fetchToken(name, segmentToSplit.getSegmentId(), context)
```

We should pass the `ProcessingContext` here.